doc/reduce-memory.md

```diff
@@ -0,0 +1,49 @@
+# Reduce Memory
+
+There are a few parameters that can be dialed down to reduce the memory usage of `bitcoind`. This can be useful on embedded systems or small VPSes.
```

Comment:
```suggestion
There are a few parameters that can be dialed down to reduce the memory usage of `dashd`. This can be useful on embedded systems or small VPSes.
```